specification/workloadmonitor/resource-manager/readme.md

```diff
@@ -59,6 +59,14 @@ directive:
 ```
 
 ---
+
+## Suppression:
+``` yaml
+directive:
+  - suppress:
+      - R4017 # HealthMonitors only applied to top-level resources.
```

Comment:
We need to scope this suppression: adding a 'from' to indicate the file it is suppressed in is fine.

Here is an example: https://github.com/Azure/azure-rest-api-specs/blob/6455851aa3d2882cb6482e5b1d76c92138b38447/specification/cosmos-db/resource-manager/readme.md

Reply:
Am I doing this wrong? The error is no longer getting suppressed.